src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -556,6 +569,11 @@ internal bool IsNetStandardFeatureFlagEnabled()
             return headerTags;
         }
 
+        internal string[] ParseStringArray(string commaSeparatedValues)
+        {
+            return commaSeparatedValues.Split(',', StringSplitOptions.RemoveEmptyEntries | StringSplitOptions.TrimEntries);
```

Comment:
`StringSplitOptions.TrimEntries` is only available in recent version of .NET (5+ I think?). If we're going to use it, we need conditional compilation (`#if NET5`). This is why my other suggestion called `String.Trim()` on each one item in the array.

Reply:
Added code to manually split and not use LINQ